documentation/staging/content/userguide/managing-operators/installation.md

```diff
@@ -245,6 +257,17 @@ Note that the Domain custom resource definition is shared.
 Do not delete the CRD if there are other operators in the same cluster
 or you have running domain resources.
 
+Beginning with Operator version 4.0, uninstalling an operator also removes the conversion webhook
```

Comment:
webhook deployment and its associated resources by default.

Reply:
Fixed.